packages/core/src/external/carequality/ihe-gateway-v2/ihe-gateway-v2-logic.ts

```diff
@@ -152,23 +188,13 @@ export async function createSignSendProcessDqRequests({
   });
 
   const resultPromises = signedRequests.map(async (signedRequest, index) => {
-    return sendProcessRetryDqRequest({
+    const result = await sendProcessRetryDqRequest({
       signedRequest,
       samlCertsAndKeys,
       patientId,
       cxId,
       index,
     });
-  });
-
-  const results = await Promise.allSettled(resultPromises);
-  const successfulResults = results
-    .filter(
-      (result): result is PromiseFulfilledResult<OutboundDocumentQueryResp> =>
-        result.status === "fulfilled"
-    )
-    .map(result => result.value);
-  for (const result of successfulResults) {
     try {
       // TODO not sure if we should retry on timeout
       await executeWithNetworkRetries(async () => axios.post(dqResponseUrl, result), { log });
```

Comment:
Same as above.

Reply:
gonna resolve these and lets continue the convo on that one thread